src/mbtest/server.py

```diff
@@ -135,16 +135,23 @@ def __enter__(self) -> "MountebankServer":
     def __exit__(self, ex_type, ex_value, ex_traceback) -> None:
         self.delete_imposters()
 
+    def _ensure_imposter_host_matches_server_host(self, imposter: Imposter) -> Imposter:
+        if imposter.host != self.host:
+            imposter.host = self.host
+        return imposter
+
     def add_imposters(self, definition: Union[Imposter, Iterable[Imposter]]) -> None:
         """Add imposters to Mountebank server.
 
         :param definition: One or more Imposters.
         :type definition: Imposter or list(Imposter)
         """
+
         if isinstance(definition, abc.Iterable):
             for imposter in definition:
                 self.add_imposters(imposter)
         else:
+            definition = self._ensure_imposter_host_matches_server_host(imposter=definition)
```

Comment:
Not sure I see the need for this. Can't we just set it? `definition.host = self.host` should work.

Reply:
I'm also seeing test coverage failures - not sure why they're not showing up here.

Reply:
> Not sure I see the need for this. Can't we just set it? `definition.host = self.host` should work.

Agreed, much simpler.

Will take a look at the coverage failures once I'm set up with Python 3.5 and 3.8.

Reply:
I use [PyEnv](https://github.com/pyenv/pyenv) for setting up various Python versions, FWIW.